ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -573,7 +573,11 @@ else
                 {
                     <div><b>Want to print?</b> <a href="@puzzleFilePath" target="_blank" download>Click this link</a> to download the file. Please allow popups!</div>
                 }
-
+                
+                @if (!Model.IsPuzzleForSinglePlayer && !Model.Event.EphemeralHackKillSync)
```

Comment:
Two things:

1) needs to check for `Model.Event.AllowBlazor` for as long as that flag actually exists in the code.
2) I am concerned that the kill switch doesn't kill sessions in progress, meaning that it won't fix anything until we badger people to refresh puzzle pages, which serves to reward people who ignore that request because they still get to co-op. Should the `ClientSyncComponent` look for the kill switch every N minutes? Doesn't need to be instantaneous, hence the use of minutes not seconds for polling.

Reply:
Alternatively, add some sort of infrastructure parallel to presence which can push this notification to all puzzle pages rather than the pages having to poll for it. Alternatively alternatively, abuse the presence infrastructure to do it somehow.

Reply:
Will handle in a followup change